10_binary_counter.md

```diff
@@ -451,8 +460,12 @@ Notice that zero is `const T&` reference because we don't plan to modify it,
 but we do modify carry,
 so it should be passed by value.
 
-The second function applies the operation
-to all the elements left sitting in the counter.
+After we finish adding all our elements to the counter, they might not all be reduced to one element.
```

Comment:
This justifies the need for the reduce_counter operation and is a quote from Alex.